src/stylesheets/HomepageRow.css

```diff
@@ -1,5 +1,5 @@
-p {
-  font-size: 19.8px;
+* {
+  font-size: max(16px);
```

Comment:
Does this not create an accessibility concern if folks can't increase the text size?

Reply:
That's true. I didn't think about that. 

I guess one approach would be to reduce the amount of content in those sections so the text doesn't spill over when the browser font size is increased.

Reply:
I was able to reduce the content and test it on large font sizes.
I am no longer getting that overlapping text issue.

Reply:
Maybe we can think of setting the size of the sections in `em` or `rem` units

Reply:
I changed it to rem